node-cli/users.js

```diff
@@ -0,0 +1,106 @@
+const inquirer = require("inquirer");
+const Realm = require("realm");
+const index = require("./index");
+const config = require("./config");
+const main = require("./main");
+const output = require("./output");
+
+const REALM_APP_ID = config.realmAppId;
+const appConfig = {
+  id: REALM_APP_ID,
+  timeout: 10000,
+};
+
+const app = new Realm.App(appConfig);
+
+authedUser = {};
```

Comment:
Couldn't you use `app.currentUser` instead of tracking this user yourself?

Reply:
+1 What if we move `app` and `Realm` logic into a new file: `realm.js`


```js
// from users.js
export const app = new Realm.App(appConfig);
// from index.js
openRealm(); // Only place we call getAuthedUser()
closeRealm();
getRealm();
```

- Let's us remove `authedUser` and `getAuthedUser()` in favor of `app.currentUser`
- Cleaner separation of concerns for index.js vs realm.js